lib/gollum/views/page.rb

```diff
@@ -265,6 +260,10 @@ def table(data)
         result << "</tr>\n</table>\n"
       end
 
+      def title
```

Comment:
Looks like making this method private is triggering test failures on `test_page_view.rb:143` where this is called:

```ruby
 actual = @view.title
```

I think you can change that to the following to get around this:

``ruby
actual = @view.send(:title)
```

Reply:
Thank you, I completely missed this.

Instead of testing the now-private `title` method I added additional tests for the public `page_header` method that is exposed on the frontend.

So now `h1_title = false` and `h1_title = true` cases are explicitly tested against `page_header` instead of `title`.